projects/Mallard/src/components/article/html/css.ts

```diff
@@ -161,7 +160,7 @@ const makeCss = ({ colors, theme }: CssProps, contentType: string) => css`
         .content-wrap {
             padding-left: ${px(metrics.article.sides)};
             padding-right: ${px(metrics.article.sides)};
-    
+
```

Comment:
Do we need changes in this file? (new imports doesn't seem to be used)

Reply:
it's just the getScaledFont got moved from `webview` to `typography`